migrate.py

```diff
@@ -173,8 +173,12 @@ def main(options):
             comments = get_issue_comments(issue['local_id'], bb_url, options.bb_auth)
 
         gh_issue = convert_issue(issue, options, gh_milestones)
-        gh_comments = [convert_comment(c, options) for c in comments
-                       if convert_comment(c, options) is not None]
+        gh_comments = [
+            convert_comment(c, options) for c in comments
+            # Bitbucket status comments (assigned, version, etc. changes) are
+            # not imported to minimize noise.
+            if c['content']
```

Comment:
this change makes sense. But can you add onto the comment something like`# These BB status comments have no content` to maximize clarity?

Reply:
done!